ESP32LapTimer/src/TimerWebServer.cpp

```diff
@@ -4,27 +4,26 @@
 #include "ADC.h"
 #include "RX5808.h"
 #include "Calibration.h"
-#include "Wireless.h"
 
 #include <esp_wifi.h>
 #include <FS.h>
 #include "TimerWebServer.h"
 #include <WebServer.h>
 #include "SPIFFS.h"
 #include <Update.h>
+#include <ESPAsyncWebServer.h>
 
-static WebServer  webServer(80);
-static WiFiClient client = webServer.client();
+AsyncWebServer webServer(80);
 
 //flag to use from web update to reboot the ESP
 //static bool shouldReboot = false;
 static const char NOSPIFFS[] PROGMEM = "You have not uploaded the SPIFFs filesystem!!!, Please install the <b><a href=\"https://github.com/me-no-dev/arduino-esp32fs-plugin\">following plugin</a></b>.<br> Place the plugin file here: <b>\"<path to your Arduino dir>/tools/ESP32FS/tool/esp32fs.jar\"</b>.<br><br> Next select <b>Tools > ESP32 Sketch Data Upload</b>.<br>NOTE: This is a seperate upload to the normal arduino upload!!!<br><br> The web interface will not work until you do this.";
 
-
-static bool firstRedirect = true;
 static bool HasSPIFFsBegun = false;
 
-static bool HTTPupdating = false;
+static bool airplaneMode = false;
```

Comment:
airplaneMode probably got left over when updating your code. It lives in Wireless.h now so can just remove it.